mostafashahat2021.github.io_2022-09-23_22-24-21.report.html

```diff
@@ -0,0 +1,266 @@
+<!--
```

Comment:
- Delete this whole file

Reply:
@Lucas-Masaba which file? I should delete?

Reply:
@MostafaShahat2021, this very file. The `report.html` file. It is an imposter 😄

Reply:
thanks a lot, @Lucas-Masaba  for your efforts, I fixed all the button errors and I got only one warning,
but still not pass the GitHub check 😭, how can I fix this warning
![error4](https://user-images.githubusercontent.com/79546797/192059942-c6b844fe-e884-4bc1-9b3a-2a1616920e24.jpg)

Reply:
@Lucas-Masaba Could not get results from HTML checker, Unexpected token < in JSON at position 0
what does that mean?

Reply:
Hello @MostafaShahat2021, 
Let me ask my colleagues if they have any solution to this error. I cannot seem to find where it is coming from.

Reply:
@Lucas-Masaba Thank you so much, I really appreciate your efforts with me today 🙂, Hope we can find a solution for this error, you know that tomorrow is my last chance to submit this project